src/Xamarin.Android.Tools.Bytecode/Tests/ParameterFixupTests.cs

```diff
@@ -63,6 +58,48 @@ public void XmlDeclaration_DoesNotThrowAnExceptionIfDocumentationNotFound ()
 				Assert.Fail ("An unexpected exception was thrown : {0}", ex);
 			}
 		}
+
+		[Test]
+		public void DocletType_ShouldDetectApiXml ()
+		{
+			string tempFile = null;
+
+			try {
+				tempFile = LoadToTempFile ("ParameterFixupApiXmlDocs.xml");
+
+				AssertDocletType (tempFile, Bytecode.JavaDocletType._ApiXml);
+			} catch (Exception ex) {
+				try {
+					if (File.Exists (tempFile))
+						File.Delete (tempFile);
+				}
+				catch { }
+
+				Assert.Fail ("An unexpected exception was thrown : {0}", ex);
```

Comment:
...all the more reason to delete `tempFile` within a `finally` block instead of a `catch` block: this `Assert` wouldn't be needed anymore.